analyses/molecular-subtyping-HGG/hgg-subset/hgg_gistic_broad_values.tsv

```diff
@@ -7,54 +7,54 @@ sample_id	Kids_First_Biospecimen_ID	1p	19q	7p	7q	10p	10q
 7316-3022	BS_79SYEHY3	0	0	0	0	0	0
 7316-3303	BS_96S0VQBN	0	0	0	0	0	0
 7316-325	BS_1YTHM07J	0	0	0	0	0	0
-7316-409	BS_B9QP40ER	0	0	0	0	0	0
+7316-409	BS_B9QP40ER	0	0	0	0	0.667	0.667
```

Comment:
@kgaonkar6, yes, the only thing I can think of is now for running GISTIC, we implemented `copy.num=ploidy` and that might have changed the results. Since OpenPBTA has not implemented the changes when I pull the results, this might be the cause of it.

Reply:
Ah ok good point, then it looks good to me